src/python/sdk/pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "econia-sdk"
-version = "0.1.0"
+version = "1.0.0"
 description = ""
-authors = ["qdrs <qdrs@proton.me>"]
+authors = ["egd <elliott@econialabs.com>"]
```

Comment:
Please include "alnoki <43892045+alnoki@users.noreply.github.com>"

@bogberry @CRBl69 what to include for you?

Reply:
![image](https://github.com/econia-labs/econia/assets/71022090/9c3aa434-2b1a-4b99-ae67-5da560f10b50)

Reply:
just discussed offline this is for point of contact

we are going to use developers@econialabs.com

Reply:
sorry for not clarifying about this earlier
i was really just thinking about moving forward how to make sure the right people get the right emails
oversight on my part

Reply:
See https://github.com/econia-labs/econia/pull/380